components/admin_console/admin_definition.jsx

```diff
@@ -5683,47 +5676,19 @@ const AdminDefinition = {
                         help_text_markdown: false,
                         placeholder: t('admin.experimental.userStatusAwayTimeout.example'),
                         placeholder_default: 'E.g.: "300"',
-                        isDisabled: it.not(it.userHasWritePermissionOnResource('experimental')),
-                    }, // eslint-disable-next-line lines-around-comment
-                    // {
-                    //     type: Constants.SettingsTypes.TYPE_BOOL,
-                    //     key: 'ServiceSettings.ExperimentalStrictCSRFEnforcement',
-                    //     label: t('admin.experimental.experimentalStrictCSRFEnforcement.title'),
-                    //     label_default: 'TODO:',
-                    //     help_text: t('admin.experimental.experimentalStrictCSRFEnforcement.desc'),
-                    //     help_text_default: 'TODO',
-                    //     help_text_markdown: false,
-                    // },
-                    // {
-                    //     type: Constants.SettingsTypes.TYPE_LIST,
-                    //     key: 'TeamSettings.ExperimentalDefaultChannels',
-                    //     label: t('admin.experimental.experimentalDefaultChannels.title'),
-                    //     label_default: 'Default Channels:',
-                    //     help_text: t('admin.experimental.experimentalDefaultChannels.desc'),
-                    //     help_text_default: 'A comma-separated list of default channels every user is added to automatically after joining a new team. Only applies to public channels, but affects all teams on the server. When not set, every user is added to `off-topic` and `town-square` channel by default. Note that even if `town-square` is not listed, every user is added to that channel after joining a new team.',
-                    //     help_text_markdown: true,
-                    //     placeholder: t('admin.experimental.experimentalDefaultChannels.example'),
-                    //     placeholder_default: 'E.g.: "channel1, channel2, off-topic"',
-                    // },
-                    // {
-                    //     type: Constants.SettingsTypes.TYPE_TEXT,
-                    //     key: 'EmailSettings.ReplyToAddress',
-                    //     label: t('admin.experimental.replyToAddress.title'),
-                    //     label_default: 'Reply To Address:',
-                    //     help_text: t('admin.experimental.replyToAddress.desc'),
-                    //     help_text_default: 'TODO',
-                    //     help_text_markdown: true,
-                    //     placeholder: t('admin.experimental.replyToAddress.example'),
-                    //     placeholder_default: 'E.g.: "reply-to@example.com"',
```

Comment:
@BenCookie95 @sbishel 
These were introduced in https://github.com/mattermost/mattermost-webapp/pull/2588 and haven't been modified or touched since. Doesn't feel right to leave commented code in the codebase for this long.

Reply:
Yeah take this out too